src/pages/appflow/destinations/cli.md

```diff
@@ -0,0 +1,31 @@
+---
+title: 'Command-line Deploy'
+previousText: 'Builds/Automations'
+previousUrl: '/docs/appflow/destinations/builds'
+nextText: 'Manual'
+nextUrl: '/docs/appflow/destinations/manual'
+---
+
+You can also create a build that will be deployed to an app store or deploy an already completed build as well.
+
+NOTE: To deploy a build you will need to have permissions to be able to deploy builds from cli.
```

Comment:
```suggestion
<blockquote>
<b>Note:</b> To deploy a build you will need to have permissions to be able to deploy builds from cli.
</blockquote>
```

Reply:
We should make our notes consistent across pages.

Reply:
for future reference, you can do blockquote and bold from markup
`>` for blockquote
and
`**` for bold